docs/graphql.rst

```diff
@@ -181,10 +173,10 @@ The agentProgram value will be used later for getting a translation into a meani
 .. code-block:: text
 
     {
-      allAgent {
+      agent(token:
+      "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiYWRtaW4iOnRydWUsImp0aSI6IjcyMzFmZjk5LTE0NDktNDRhMS04YzE2LTY4OTMzNjgwZTU4YSIsImlhdCI6MTU5OTQyMjk5MywiZXhwIjoxNTk5NDI2NTkzfQ.UP1FSU3hNOI6EiEt2sMJ4V5n2BN3ttICsJFhJXJTowU"){
```

Comment:
Trim the token length so that it fits within 80 characters. Easier readability. For example.

`eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI... ...J4V5n2BN3ttICsJFhJXJTowU`

There are a number of repeats, so this would be an easy fix.